docs/experiments-msmarco-passage.md

```diff
@@ -103,13 +103,16 @@ For that we first need to convert the run file into TREC format:
 ```bash
 $ python tools/scripts/msmarco/convert_msmarco_to_trec_run.py \
    --input runs/run.msmarco-passage.bm25tuned.txt --output runs/run.msmarco-passage.bm25tuned.trec
+$ python tools/scripts/msmarco/convert_msmarco_to_trec_qrels.py \
+   --input tools/topics-and-qrels/qrels.msmarco-passage.dev-subset.txt --output tools/topics-and-qrels/qrels.msmarco-passage.dev-subset.trec
```

Comment:
this file is already in the repo: https://github.com/castorini/anserini-tools/blob/master/topics-and-qrels/qrels.msmarco-passage.dev-subset.txt

So I think the solution is to retain this command, but write the updated qrels to `collections/msmarco-passage/qrels.dev.small.trec` - and leave the command below unchanged.

Can you verify that the two files are exactly the same?

Reply:
Yeup I have verified that the contents are the same (though they have different format, but after removing all spaces they are exactly the same)

Reply:
Okay, waiting on this fix, then we can merge everything all at once.